### DISCUSSION TITLE
Invalid REFERENCE_WORLD_SIZE in config!

### DISCUSSION THREAD

**Discussion by Farhana005** [jinfagang/yolov7#115] [Q&A]
I'm getting this error "AssertionError: Invalid REFERENCE_WORLD_SIZE in config!" while trying to train. would you help plz? @jinfagang

**Comment by Hommoner** [comment]
try add this one,
cfg.SOLVER.REFERENCE_WORLD_SIZE = 0

**Comment by Loayaltal** [comment]
change number of workers to 4 instead of 1. Will fix this.
